recipes/vollseg-napari/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "vollseg-napari" %}
+{% set version = "2.3.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/vollseg-napari-{{ version }}.tar.gz
+  sha256: bc1a435f3fbd08a38f5a6fae0b7d7581ee91e56e46f2ae933f59f1d0de8fe57f
+
+build:
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

Comment:
Please convert this recipe to `noarch: python` or explain why it doesn't qualify. (missing dependencies on a platform (Windows) are fine for noarch: python).